sound/soc/sof/intel/hda-dai.c

```diff
@@ -617,6 +617,10 @@ static int hda_dai_suspend(struct hdac_bus *bus)
 			sdai = swidget->private;
 			ops = sdai->platform_private;
 
+			if (rtd->dpcm[hext_stream->link_substream->stream].state !=
+			    SND_SOC_DPCM_STATE_PAUSED)
+				continue;
+
```

Comment:
nitpick: repeated 'the' in commit message (it also ends up cleaning up ~~the~~ the DMA)